### DISCUSSION TITLE
x-mask leading zeros

### DISCUSSION THREAD

**Discussion by jensolafkoch** [alpinejs/alpine#2864] [7. Bugs (archived)]
If you enter a number with leading zeros, those are formatted as well, like:

000,001.99

Not _exactly_ a bug and nothing really important - but I guess removing all but one digit would be a nicer formatting?

**Comment by SimoTod** [comment]
I think it makes sense for the $money helper but other masks might need leading zeros or might not even number so I wouldn't make it a general rule.

**Reply by jensolafkoch** [reply]
Yes, you are right, I meant the $money helper of course!
